extensions/cdc-debezium/src/test/java/com/hazelcast/jet/cdc/DebeziumCdcIntegrationTest.java

```diff
@@ -437,6 +438,42 @@ public void notFailWhenOldValueNotPresent() {
             assertTrueEventually(() -> assertThat(hz.getList("notFailWhenOldValueNotPresent")).isNotEmpty());
         }
     }
+    @Test
+    public void noFailWhenNoPrimaryKey() throws Exception {
+        try (PostgreSQLContainer<?> container = postgresContainer()) {
+            container.start();
+            //when
+            try (Connection connection = getPostgreSqlConnection(container.getJdbcUrl(), container.getUsername(),
+                    container.getPassword())) {
+                connection.setSchema("inventory");
+                Statement statement = connection.createStatement();
```

Comment:
Statement should be closed.  
try (Statement statement = connection.createStatement()) {...}

Reply:
Good catch! https://github.com/hazelcast/hazelcast/pull/23768/commits/c63631cb0f142faab8c14808bc1a9b34af0d6be3